app/models/admins.rb

```diff
@@ -15,4 +15,9 @@ def self.github_ids
   def self.verify?(uid)
     new.verify?(uid)
   end
+
+  def self.verify_or_fail!(uid)
+    fail UnauthorizedError::MustBeAdmin unless verify?(uid)
+    true
+  end
```

Comment:
Also here and in the other authenticators if you don't have the return statement, you don't have to worry about returning true